### ISSUE TITLE
Binance-Trader-2

### ISSUE THREAD

**Issue by MilesFrampton** [MilesFrampton/BinanceTrader#3]
Create an API call to Binance to execute a sell.

Assigned to: Martin

DoD: Integration test showing that a request from the UI to start trading causes a sell when the sell level has been reached.

**Comment by Vuologist** [comment]
I don't think we can start on this until we get a fake account up and running. Otherwise testing can become very expensive.
